src/unix/linux/disk.rs

```diff
@@ -394,7 +394,7 @@ fn get_all_list(container: &mut Vec<Disk>, content: &str, refresh_kind: DiskRefr
                 => true,
                 "tmpfs" => !cfg!(feature = "linux-tmpfs"),
                 // calling statvfs on a mounted CIFS or NFS may hang, when they are mounted with option: hard
-                "cifs" | "nfs" | "nfs4" => !cfg!(feature = "linux-netdevs"),
+                "cifs" | "nfs" | "nfs4" | "autofs" => !cfg!(feature = "linux-netdevs"),
```

Comment:
The comment needs to be updated then.

Reply:
Updated.

Reply:
Yes it happened on one of my device.

> autofs is a program for automatically mounting directories on an as-needed basis. Auto-mounts are mounted only as they are accessed, and are unmounted after a period of inactivity. Because of this, automounting NFS/Samba shares conserves bandwidth and offers better overall performance compared to static mounts via fstab.

In my understanding it's a wrapper for NFS/Samba, so I added the comment: `or through autofs`.

Reply:
:+1: